array/README.md

```diff
@@ -0,0 +1,45 @@
+<p align="Center">
+  <img src="https://bixbydevelopers.com/dev/docs-assets/resources/dev-guide/bixby_logo_github-11221940070278028369.png">
+  <br/>
+  <h1 align="Center">Arrays and Modeling</h1>
+</p>
+
+**Purpose**
+
+The purpose of this sample capsule is to demonstrate how Bixby actions and views handle array of objects (both primitive and structured concepts)
+
+  - No actual API calls, only fixed lookup in /code/lib/lib.js
+  - max (One) or max (Many) determines the argument type in linked JS 
```

Comment:
I am not sure if I understand, what is a linked JS? maybe best to have someone from docs team proof read the readme.

Reply:
Yes, definitely requires DOC's help on the README.md

What I meant by linked JS, is the Javascript file linked to the action in endpoints files.

Reply:
Ok, I trust that you will run it by them later